cypress/e2e/session-recording.cy.ts

```diff
@@ -280,6 +280,62 @@ describe('Session recording', () => {
                     )
                 })
             })
+
+            it('it captures XHR method correctly', () => {
```

Comment:
these tests have a lot of duplication now, but I want to get this fix out

Reply:
This test is pretty convincing, how would it have failed before?

Reply:
The method = POST assertion?

Reply:
yep... it was always GET before (or at least overwhelmingly GET I didn't check which)

Reply:
and for clarity I validated the test failed before the change 😊